core/de.json

```diff
@@ -5,7 +5,7 @@
         "back": "Zurück zu Muonium",
         "favorites": "Favoriten",
         "login": "Login",
-        "notready": "Geschlossene Beta, willkommen zum Königreich der Wale",
+        "notready": "Private Beta, willkommen im Königreich der Wale",
```

Comment:
"Private Beta" is also German, and "geschlossene Beta" would have meant something like closed Beta, that's why I changed it